site_scons/site_tools/roc/commands.py

```diff
@@ -103,6 +103,34 @@ def Ragel(env, source):
 
     return [env.Object(target)]
 
+def GeneratePkgConfig(env, build_dir, prefix, libdir, filename, version, dependencies, libs):
+    output_dir = env.Dir(build_dir).path
+    target = os.path.join(output_dir, filename)
+    src = """prefix=%(prefix)s
+exec_prefix=${prefix}
+libdir=%(libdir)s
+includedir=${prefix}/include
+
+Name: libroc
+Requires: %(dependencies)s
+Version: %(version)s
+Description: Real-time audio streaming over the network.
+URL: https://roc-project.github.io
```

Comment:
Nitpick: I'm trying to keep our scons plugin (the code under site_scons/site_tools/roc) generic, keeping project-specific stuff in SConstruct and SConscript. So let's pass Name, Description, and URL as arguments instead of hard-coding them.